### PR TITLE
Test graph ql

### PR DESCRIPTION
**Pull request by TomJKing**
Merge in graph ql code to master branch

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -8,6 +8,11 @@ lazy val root = (project in file(".")).enablePlugins(PlayScala)
 scalaVersion := "2.12.8"
 
 resolvers += "Atlassian Releases" at "https://maven.atlassian.com/public/"
+resolvers += Resolver.bintrayRepo("jarlakxen", "maven")
+
+resolvers += "Apollo Bintray" at "https://dl.bintray.com/apollographql/maven/"
+resolvers += "Acccc" at "https://mvnrepository.com/artifact/"
+resolvers += Resolver.jcenterRepo
```
Comment (suzannehamilton) [inline build.sbt]: Do we need all these resolvers to add drunk, or are some of them left over from other spikes?

File: app/graphQL/TDRGraphQLClient.scala [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+import com.github.jarlakxen.drunk._
+
+import scala.collection.immutable.Seq
+import scala.concurrent.Future
+
+object TDRGraphQLClient {
+
+  implicit val system = ActorSystem()
+  implicit val materializer: ActorMaterializer = ActorMaterializer()
+  val uri: Uri = Uri(s"https://qad2wpgi3befniyihgl42yvfea.appsync-api.eu-west-2.amazonaws.com/graphql")
```
Comment (suzannehamilton) [inline app/graphQL/TDRGraphQLClient.scala:20]: At some point soon we should add this to application.conf in an environment variable, so we can use different endpoints in different environments.

It's probably fine to leave it for now, unless it'll make your Terraforming easier?

**Review by suzannehamilton — APPROVED** [review]
A couple of very minor comments, but I think this is worth merging into the prototype. :+1:

File: app/graphQL/TDRGraphQLClient.scala (hunk 2 above)
Comment (TomJKing) [reply app/graphQL/TDRGraphQLClient.scala:20]: Added comment that this should be moved to the application configuration at some point

File: build.sbt (hunk 1 above)
Comment (TomJKing) [reply build.sbt]: Removed resolvers

**Pull request merged**